client/service/examples/logger/logger/logger_test.go

```diff
@@ -71,21 +70,12 @@ func TestClose(t *testing.T) {
 	logger := New(s)
 
 	go server.Start()
-
-	var wg sync.WaitGroup
-
-	wg.Add(1)
-	go func() {
-		defer wg.Done()
-		assert.Nil(t, logger.Start())
```

Comment:
I think this check should stay because we're testing if _Start()_ will return an err or not